components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/starknet_architecture_overview.adoc

```diff
@@ -0,0 +1,258 @@
+Operating your own node is typically faster than using a shared architecture, like the
+gateway.
+
+It’s worth noting that more people running nodes increases
+the resilience of the network and prevents server flooding, which has
+been an issue in other L2 networks.
+
+Currently, there are primarily three methods for a node to keep track of
+the network’s state and nodes can implement any of these
+methods:
+
+. *Replaying Old Transactions*: Like Ethereum or Bitcoin, a node can
+take all the transactions and re-execute them. Although this approach is
+accurate, it isn’t scalable unless you have a powerful machine that’s
+capable of handling the load. If you can replay all transactions, you
+can become a sequencer.
+
+. *Relying on L2 Consensus*: Nodes can trust the sequencer to
+execute the network correctly. When the sequencer updates the state and
+adds a new block, nodes accept the update as accurate.
+
+. *Checking Proof Validation on L1*: Nodes can monitor the state of
+the network by observing L1 and ensuring that every time a proof is
+sent, they receive the updated state. This way, they don’t have to trust
+anyone and only need to keep track of the latest valid transaction for
+Starknet.
+
+Each type of node setup comes with its own set of hardware requirements
+and trust assumptions.
+
+=== Nodes that replay transactions
+
+Nodes that replay transactions require powerful machines to track and
+execute all transactions. These nodes don’t have trust assumptions; they
+rely solely on the transactions they execute, guaranteeing that the
+state at any given point is valid.
+
+=== Nodes that rely on L2 consensus
+
+Nodes that rely on L2 consensus require less computational power. They
+need sufficient storage to keep the state but don’t need to process a
+lot of transactions. The tradeoff here is a trust assumption.
+Currently, Starknet revolves around one sequencer, so these nodes
+trust StarkWare not to disrupt the network. However, once a consensus
+mechanism and leader election amongst sequencers are in place, these
+nodes will only need to trust that a sequencer who staked their stake to
+produce a block is not willing to lose it.
+
+=== Nodes that check proof validation on L1
+
+Nodes that only update their state based on proof validation on L1
+require the least hardware. They have the same requirements as an
+Ethereum node, and once Ethereum light nodes become a reality,
+maintaining such a node could be as simple as using a smartphone. The
+only tradeoff is higher latency. Proofs are sent to Ethereum intermittently, not for every block, resulting in delayed state updates. Plans are in
+place to produce proofs more frequently, even if they are not sent to
+Ethereum immediately, lowering node latency.
+
+// == Conclusion
```

Comment:
I feel like a Conclusion here is not necessary and doesn't really add any value. In general, it's more appropriate for a Book that you'll read beginning to end, and doesn't fit the style of the docs, at least not where we're placing it.